client/layout/index.jsx

```diff
@@ -299,9 +299,6 @@ class Layout extends Component {
 				{ config.isEnabled( 'legal-updates-banner' ) && (
 					<AsyncLoad require="calypso/blocks/legal-updates-banner" placeholder={ null } />
 				) }
-				{ config.isEnabled( 'nav-unification' ) && ! config.isEnabled( 'jetpack-cloud' ) && (
```

Comment:
@cpapazoglou I didn't move the condition `! config.isEnabled( 'jetpack-cloud' )` to the new location for this modal. Do you think this should be in there too?

Reply:
Nope, since `client/my-sites/sidebar-unified/index.jsx` is not rendered for `jetpack-cloud`. Even if it did, it would mean that it would have new UI, os it would deserve the modal 😜!